marissahuysentruyt/estimate-tracker
# Feat: Build basic inputs

## Description


This PR adds support for 4 basic inputs within a form.
- new basic inputs
- submit button
- oklch colors
- HomeView, DetailsView tests updated

Although out of scope, this PR adds support for OKLCH colors.


Closes #5 




## To Validate



1. Make sure all PR Checks have passed
2. Pull down this branch
3. Run `npm install` for new dependencies
4. Run `npm run test` and `npm run test:e2e` to verify all tests pass
5. Run `npm run lint` to verify there are no linting errors
6. Run `npm run dev` and visit the [homepage](http://localhost:5173/)
7. Try to submit the estimate tracker's form with the "Save" button. You should receive an error and be asked to fill out the inputs
8. Input some dummy info for the 4 inputs and submit the form. You should see the data appear as a list below the form like so:

<img width="302" alt="Screenshot 2024-05-15 at 12 20 45 PM" src="https://github.com/marissahuysentruyt/estimate-tracker/assets/69602589/a4833a76-f4a8-49a3-8ec8-78de83920340">

9. Verify there are no console errors in dev tools
